test/paper_toggle_button_test.dart

```diff
@@ -0,0 +1,83 @@
+@TestOn('browser')
+library polymer_elements.test.paper_toggle_button_test;
+
+import 'dart:async';
+import 'package:polymer_elements/paper_toggle_button.dart';
+import 'package:web_components/web_components.dart';
+import 'package:test/test.dart';
+import 'common.dart';
+
+/**
+ * Original tests:
+ * https://github.com/PolymerElements/paper-toggle-button/tree/master/test
+ */
+
+main() async {
+  await initWebComponents();
+
+  group('defaults', () {
+    PaperToggleButton b1;
+
+    setUp(() {
+      b1 = fixture('basic');
+    });
+
+    test('check button via click', () {
+      Completer done = new Completer();
+
+      b1.on['click'].take(1).listen((_) {
+        expect(b1.getAttribute('aria-checked'), equals(isTrue));
+        expect(b1.checked, isTrue);
+        done.complete();
+      });
+
+      tap(b1);
+
+      return done.future;
+    }, skip: 'https://github.com/dart-lang/polymer_elements/issues/47');
```

Comment:
Mentioned it in the issue, but it looks like `aria-pressed` is the correct attribute for toggle buttons. Mind just updating the test to check that attribute?